docs/guides/seeder-user-guide.md

```diff
@@ -23,93 +21,133 @@ dig dns-introducer.chia.net
+
+In `config.yaml` the `domain_name` is `my-chia-seeder.example.com.` and the `nameserver` is `vps.example.com.`. Note the trailing periods. The main thing to change in `soa` is adding a correct contact email in `rname` and optionally changing the `serial_number`.
 
 ## Running
 
 ```bash
-. ./activate
-chia start seeder crawler
+$ . ./activate
+$ chia start seeder
 ```
-
 will run both a crawler and a DNS server.
 Alternatively,
+```bash
+$ . ./activate
+$ chia start crawler
+```
+gives you the option to merely crawl the network so as to get a list of the connectable nodes, without having to set up a DNS server.
+
+## Testing
+
+You should test to make sure that the Chia node can connect. 
+
+On a version of chia-blockchain newer than 1.6.2, stop node then delete `peers.dat` in `~/.chia/mainnet/db/`. 
+
+Modify your `config.yaml` to include only your dns seeder. 
+
+If you tail the logs as you start up the node you should see the node connect and obtain both IPv4 and IPv6 nodes. 
 
 ```bash
-. ./activate
-chia start crawler
+2023-01-14T15:19:30.969 full_node chia.full_node.full_node: INFO     Received 32 peers from DNS seeder, using rdtype = A.
+2023-01-14T15:19:31.010 full_node chia.full_node.full_node: INFO     Received 32 peers from DNS seeder, using rdtype = AAAA.
 ```
 
-gives you the option to merely crawl the network so as to get a list of the connectable nodes, without having to set up a DNS server.
+Sometimes dns takes a moment so seeing some initial flakiness here is not cause for concern. Just stop again, delete `peers.dat` and try the test again.
 
 ## Stopping
 
 ```bash
-. ./activate
-chia stop -d all
+$ . ./activate
+$ chia stop -d all
 ```
+
+## Being included in config.yaml
+
+There are a couple of criteria we all look for when adding a seeder to the initial config file.
+
+You must have an ICANN registered domain name. Your seeder host must support IPv6 and IPv4.
+
+We ask that you commit to a monthly uptime of 99.9% which is available enough to be reliable but also leaves flexibility to not need to run a cluster and gives you time for a reboot once in a while to keep up with things like security patches. We will be monitoring all of the seeders listed in the most recent version of `initial-config.yaml`. We highly recommend that you enable monitoring of your seeder as well. We are heavy users of [Uptime Robot](https://uptimerobot.com/) but anything similar will do.
+
+Being a known community member - the better you're known (pseudonymously is fine) - the more likely you are to be added. You should be on [Keybase](https://keybase.io/team/chia_network.public) and we will need to know your Keybase handle as you will be added to a seeder operator's closed channel.
```

Comment:
Let's not add a new keybase reference for poor Roy to remove later :)

Reply:
Having Keybase here is intentional. We don't want to do security conversations on Discord

Reply:
I've updated the KB instructions in that case